libs/data/src/routing/routes.ts

```diff
@@ -5,9 +5,9 @@ const apiUrlPrefix = Platform.OS === 'android' ? 'http://10.0.2.2:3333/api' : 'h
 
 export const apiRoutes = {
   auctions: `${apiUrlPrefix}/auctions`,
-  auctionsByCategory: `${apiUrlPrefix}/auctions/category/:id`,
-  auction: `${apiUrlPrefix}/auctions/:id`,
-  myAuctions: `${apiUrlPrefix}/auctions/my/:id`,
+  auctionsByCategory: `${apiUrlPrefix}/auctions?categoryId=:id`,
+  auction: `${apiUrlPrefix}/auctions?auctionId=:id`,
```

Comment:
Could it be just /auctions/:id ?

Reply:
but then it will be inconsistent to other query params

Reply:
endpoint on the backend should work already, it uses findOne method